packages/dart/sshnoports/bundles/shell/systemd/sshnpd.service

```diff
@@ -2,24 +2,47 @@
 Description=Ssh No Ports Daemon
 After=network-online.target
 
-# Make this unit fail if sshd isn't started first
-Requisite=sshd.service
+# Uncomment the following line to make this unit fail if sshd isn't started first
+; Requisite=sshd.service
```

Comment:
Both # and ; make valid comments, so I used `;` to represent items that can be uncommented